LTHPasscodeViewController/LTHPasscodeViewController.h

```diff
@@ -314,6 +314,13 @@
  */
 - (void)showLockScreenWithAnimation:(BOOL)animated withLogout:(BOOL)hasLogout andLogoutTitle:(NSString*)logoutTitle;
 /**
+ @brief				Used for displaying the lock. The passcode view is added directly on the keyWindow.
+ @param superview   The superview where is to be presented, used to measure the center of the view.
+ @param hasLogout   Set to @c YES for a navBar with a Logout button, set to @c NO for no navBar.
+ @param logoutTitle The title of the Logout button.
```

Comment:
It looks OK in fullscreen and over the popup, even keeping one popup opened and going back to the main app to view it fullscreen (and back again). The rotation also works OK. (The screenshots are from MEGA app itself and the extension opened within Google Drive).
![screen shot 2017-06-09 at 9 47 01](https://user-images.githubusercontent.com/28537210/26966235-89f8614c-4cf9-11e7-8a9a-b08421f84a12.png)
![screen shot 2017-06-09 at 9 47 09](https://user-images.githubusercontent.com/28537210/26966236-89fb16c6-4cf9-11e7-9bdc-83a59c65a8a8.png)
![screen shot 2017-06-09 at 9 47 18](https://user-images.githubusercontent.com/28537210/26966237-89ff1028-4cf9-11e7-9057-eaa085c4bd49.png)
![screen shot 2017-06-09 at 9 47 29](https://user-images.githubusercontent.com/28537210/26966238-8a006af4-4cf9-11e7-8fe0-f7014591bf2f.png)